skyportal/handlers/api/observation.py

```diff
@@ -443,10 +447,12 @@ def get_observations(
             )
             query_area = sa.select(area).filter(
                 LocalizationTile.localization_id == localization.id,
+                LocalizationTile.probdensity >= min_probdensity,
```

Comment:
This seems like a good way to make things go faster. I assume you know what you are doing in terms of the logic of the query...